pkg/transforms/policy.go

```diff
@@ -80,17 +82,15 @@ func getPolicyCommonProperties(c *unstructured.Unstructured, node Node) Node {
 	return node
 }
 
-// For cert, config policies.
-func StandalonePolicyResourceBuilder(c *unstructured.Unstructured) *PolicyResource {
-	node := transformCommon(c) // Start off with the common properties
-
-	return &PolicyResource{node: getPolicyCommonProperties(c, node)}
+type StandalonePolicyResource struct {
+	node   Node
+	policy unstructured.Unstructured
```

Comment:
I think this will keep the entire policy resource in-memory.
The goal of the transforms is to extract only the min data needed so we can release the memory used by the full resource object. Unfortunately we have this pattern in other places, but I don't want to keep adding more.

I'm thinking that the best solution is to change node.Metadata to a `map[string]interface{}` so we aren't limited to only strings. We'll have to see how big is that refactor.

Reply:
It didn't end up being too bad of a refactor - adjusting `GetMetadata` made some things more convenient.